lets-update.fifthtry.site/FASTN.ftd

```diff
@@ -5,6 +5,7 @@ system: lets-update
 system-is-confidential: false
 
 -- fastn.dependency: design-system.fifthtry.site
+provided-via: lets-update.fifthtry.site/ds
```

Comment:
This line should be in template package. This package is not main package. Only main packages provided via matters. You can use this for proving things in storybook tho. For now since your ds.ftd is empty it's any ways a no op.